app.js

```diff
@@ -0,0 +1,63 @@
+window.addEventListener('DOMContentLoaded', () => {
+  const bookSection = document.querySelector('.book-list');
+  const bookTitle = document.querySelector('#title');
+  const bookAuthor = document.querySelector('#author');
+  const addBtn = document.querySelector('#submit');
+
+  const bookObjects = JSON.parse(localStorage.getItem('book-collection')) || [];
+
+  function createBook() {
+    bookSection.innerHTML = '';
+    for (let i = 0; i < bookObjects.length; i += 1) {
+      const bookContainer = document.createElement('div');
+      const title = document.createElement('h2');
+      const author = document.createElement('h2');
+      const deleteBtn = document.createElement('button');
+      deleteBtn.setAttribute('class', 'deletebtn');
+      deleteBtn.setAttribute('data', i);
+
+      title.textContent = bookObjects[i].title;
+      author.textContent = bookObjects[i].author;
+      deleteBtn.textContent = 'Delete';
+
+      bookContainer.appendChild(title);
+      bookContainer.appendChild(author);
+      bookContainer.appendChild(deleteBtn);
+      bookSection.appendChild(bookContainer);
+    }
+    /* eslint-disable-next-line no-use-before-define */
+    deleteBook();
+  }
+  function deleteBook() {
+    [...document.querySelectorAll('.deletebtn')].forEach((element) => {
```

Comment:
- [ ] [Optional] You don't need the spread operator in the square brackets, you can rewrite this code by applying this suggestion below:

```suggestion
    document.querySelectorAll('.deletebtn').forEach((element) => {
```